lib/oli_web/controllers/pow.ex

```diff
@@ -11,23 +11,37 @@ defmodule OliWeb.PowController do
   @ttl :timer.minutes(24 * 60)
   @cache_config {PowResetPassword.Store.ResetTokenCache, ttl: @ttl}
 
-  def create_user_password_reset_link(%{"id" => id}) do
-    %{email: email} = _user = Repo.get(User, id)
+  @doc """
+  Create a password reset link for a user or author that will expire in
+  the amount of time defined in @ttl (currently 24 hours).
+
+  id: the id of the user or author
+  account_type: :user or :author
+  """
+  def create_password_reset_link(%{"id" => id}, account_type) do
```

Comment:
I noticed that `send_user_password_reset_link` and `send_author_password_reset_link` are almost identical, except for the model type. It might be interesting to see whether they can be merged into a single function call, although this part will likely be replaced with the upcoming new sign-in system.

Reply:
Yes, I agree. I didn't want to make more reactors since Pow me be removed from the codebase